multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -356,7 +356,7 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: 7eb0f1e65f7eb3e35b0739d6064b92b7621af0f4e41813c35bfdee71ceaefbe2
+        checksum/config: 
```

Comment:
Is it correct to use empty?

Reply:
No. I found that in MACOS, `sha256sum` used in `multicluster/hack/update-checksum.sh` is not available. I have regenerated the manifest files with alternative `shasum -a 256`.